pipelines/tekton/aiedge-e2e/tasks/skopeo-copy.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
@Sara4994 I think it would be useful to split this into two commits: the first one has everything here, except this `skopeo-copy.yaml` file is identical to upstream; the second has your custom changes to this file. This will make it easier to review the differences, and give you an isolated commit equivalent to what you'll eventually propose upstream.

What do you think?

Reply:
yep that makes sense... have made the changes :) thanks @grdryn